### ISSUE TITLE
cannot create instances of data structures without using new operator

### ISSUE THREAD

**Issue by rongierlach** [philpl/extendable-immutable#14]
Immutablejs at the time of this post allows for creating immutable instances with factory functions like so:  
```javascript
import { Map } from 'immutable'

const myData = Map()
// Map {}
```
In extendable-immutable not so much however:
```javascript
import { Map } from 'extendable-immutable'

const myData = Map()
// TypeError: this.__wrapImmutable is not a function
```

This however works fine:
```javascript
import { Map } from 'extendable-immutable'

const myData = new Map()
// Map {}
```

If this is functionality you're willing merge I'll get going on a pull request :)

**Comment by philpl** [comment]
Does #15 look good to you?

**Comment by rongierlach** [comment]
beautiful!

**Comment by kitten** [comment]
Published in v1.3.3
